spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/autoconfig/TraceAutoConfiguration.java

```diff
@@ -107,6 +119,9 @@ Tracing tracing(
 		for (FinishedSpanHandler finishedSpanHandlerFactory : this.finishedSpanHandlers) {
 			builder.addFinishedSpanHandler(finishedSpanHandlerFactory);
 		}
+		for (TracingCustomizer customizer : this.tracingCustomizers) {
```

Comment:
ps without tests this is subtle. it is important these apply last as they can rejig the reporter config, something needed for secondary sampling to work. ex we want to flip default reporter to noop but reuse the normal reporter as a finished span handler. this stuff works when customizes hit last before build